index.html

```diff
@@ -1,229 +1,360 @@
+      />
+      <img
+        src="./images/image-geometry_6.png"
+        alt="geometry-image-six"
+        id="geometry-image-six"
+      />
+      <div class="contact-form-info-container">
+        <h2 class="title-text">Get started</h2>
+        <p class="descriptive-text" id="contact-description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat.
+        </p>
+        <form
+          action="https://formspree.io/f/mqkobewg"
+          method="post"
+          class="contact-form"
+        >
+          <div class="name-email-text-label-box">
+            <div class="input-and-text-container" id="name-label-input-id">
+              <label for="name">Your name</label>
+              <input
+                type="text"
+                name="name"
+                id="name"
+                maxlength="30"
+                required
+                placeholder="Enter your name here.."
+              />
+            </div>
+            <div class="input-and-text-container" id="email-label-input-id">
+              <label for="email">Your email</label>
+              <input
+                type="email"
+                name="email"
+                id="email"
+                required
+                placeholder="Enter your email here.."
+              />
+            </div>
+          </div>
+          <label for="message" id="message-label-id">Message</label>
+          <textarea
+            name="message"
+            id="message"
+            placeholder="Enter your message here.."
+            maxlength="500"
+            required
+          ></textarea>
+
+          <button class="primary-button">start collaboration</button>
+        </form>
+      </div>
+      <img
+        src="./images/image-geometry_7.png"
+        alt="geometry-image-seven"
+        id="geometry-image-seven"
+      />
+    </section>
+    <footer class="footer-section">
+      <a class="get-resume-btn" href="#">
+        Get my Resume <img src="./images/ic_link_teal.png" alt="" />
+      </a>
+      <ul class="footer-social-links">
+        <li>
+          <a href="#"
+            ><img src="./images/ic_stackoverflow.png" alt="stackoverflow-img"
+          /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./images/ic_github.png" alt="github-img" /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img src="./images/ic_twitter.png" alt="twitter-img"
+          /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./images/linkedin.png" alt="linkedin-img" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./images/ic_medium.png" alt="medium-img" /></a>
+        </li>
+        <li>
+          <a href="#"><i class="fa fa-angellist" id="icon-footer"></i></a>
+        </li>
```

Comment:
- [ Optional ] Please consider adding links to your social media pages such as `GitHub` `Twitter` `LinkedIn` `Medium` and `Angelist` these links will help other online users such as recruiters to connect with you online. Currently, your `href` attributes are empty which means the links don't lead to your social media pages.

Reply:
Thank you so much. I will do that.